test/Test/Pos/Types/BlockSpec.hs

```diff
@@ -40,7 +40,7 @@ spec = describe "Block properties" $ do
     describe "verifyHeader" $ do
         prop verifyHeaderDesc (validateGoodMainHeader @SscNistBeacon)
         prop verifyHeaderDesc (validateGoodMainHeader @SscGodTossing)
-    describe "verifyHeaders" $ do
+    describe "verifyHeaders" $ modifyMaxSuccess (const 1) $ do
```

Comment:
Agreed. Alternatively one could just add `makeSmall` to the `Arbitrary` instances, but this works better-